neutromeratio/constants.py

```diff
@@ -23,6 +23,10 @@
 # ANI-1 units and conversion factors
 ani_distance_unit = unit.angstrom
 hartree_to_kJ_mol = 2625.499638
+kJ_mol_to_kT = 0.40090737504650614
+kT_to_kJ_mol = 1/0.40090737504650614
```

Comment:
warning: won't update if `kJ_mol_to_kT` is changed

Reply:
same here

Reply:
Is there a way you can use `simtk.unit` or `pint` to compute these to avoid having hard-coded constants?

Reply:
constants are now computed using simtk.unit where possible!